qemu/tests/cfg/thin_provisioning.cfg

```diff
@@ -0,0 +1,41 @@
+- thin_provisioning:
+    no Host_RHEL.m4, Host_RHEL.m5, Host_RHEL.m6
+    only Linux
+    type = thin_provisioning
+    start_vm = no
+    kill_vm = yes
+    disk_size = 1024
+    mount_point = "/home/test"
+    fstrim_cmd = "fstrim ${mount_point}"
+    post_command = "modprobe -r scsi_debug"
+    rewrite_disk_cmd = "dd if=/dev/zero of=DISK bs=1M"
+    test_image = scsi_debug
+
+    variants:
+        - unmap:
+            pre_command = "modprobe -r scsi_debug;modprobe scsi_debug dev_size_mb=${disk_size} lbpu=1"
+        - writesame:
+            pre_command = "modprobe -r scsi_debug;modprobe scsi_debug dev_size_mb=${disk_size} lbpws=1"
+    variants:
+        - passthrough_scsi:
+            mount_disk_cmd = "mkdir -p ${mount_point} && mount -o discard DISK1 ${mount_point}"
+            drive_format_scsi_debug = "scsi-hd"
+
+        - negative:
+            match_str = "the discard operation is not supported"
+            mount_disk_cmd = "mkdir -p ${mount_point} && mount -o discard DISK1 ${mount_point}"
+            drive_format_scsi_debug = "virtio"
+
+    variants:
+        - xfs:
```

Comment:
xfs and ext4 is common, pls update the a special name, otherwise it may be conflict with other cfg